collect_app/src/main/res/layout/form_entry.xml

```diff
@@ -16,7 +16,8 @@ the License.
     xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/llParent"
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
+    android:layout_height="match_parent"
+    android:importantForAutofill="noExcludeDescendants">
```

Comment:
I think it'd be better to only do this for Android 10 to minimize disruption to people who might use autofill in form entry - I don't imagine it's common, but I find it hard to discount entirely. It looks like there is a `View#setImportantForAutofill` so this should be possible.

Reply:
I'm not sure if it will be better but ok we can do that just for android 10.